pybliometrics/scopus/tests/test_SerialTitle.py

```diff
@@ -7,11 +7,15 @@
 
 from pybliometrics.scopus import SerialTitle
 
+import datetime
```

Comment:
The import order is: base modules, third party modules, own modules. Therefore, datetime should be on top